frontend/public/components/utils/kebab.tsx

```diff
@@ -484,7 +483,7 @@ export const ResourceKebab = connectToModel((props: ResourceKebabProps) => {
       isDisabled={
         isDisabled !== undefined ? isDisabled : _.get(resource.metadata, 'deletionTimestamp')
       }
-      hoverMessage={hoverMessage ? hoverMessage : t('public~Resource is being deleted.')}
+      hoverMessage={_.get(resource.metadata, 'deletionTimestamp') ? hoverMessage : null}
```

Comment:
No tooltip will be displayed if no hover message is passed down.

Reply:
I chose to not display any text over a default text as it may very well give rise to corner cases and unintentionally show the message even if the parent component had no reason to do so (i.e., didn't supply a `hoverMessage`).

Reply:
Wait, won't this hide all of the existing hover messages that were displayed? That goal was to have this logic in one place so not all callers need to supply a message.

Reply:
Added the default message back in: https://github.com/openshift/console/pull/11860/commits/8eebed347ba9fc30102efe5101a0492997898857.

Reply:
Done: https://github.com/openshift/console/pull/11860/commits/1790c8c3cf39152451b16489c6b5e5ada4409a79.